src/ui/features/onboarding/Create/Create.tsx

```diff
@@ -51,6 +46,12 @@ export function CreatePassword({
     },
   });
 
+  const { sessionDataIsLoading } = useOnboardingSession('session-expired');
```

Comment:
This just doesn't make sense if you read it :)

Reply:
I updated it to use a named parameters object and renamed `navigateOnAccountCreated` to `navigateOnExistingUser`. I know this isn't a perfect solution, and I might come up with something better later (so, this isn't the final version)